modules/mono/editor/GodotTools/GodotTools/Utils/OS.cs

```diff
@@ -106,7 +106,11 @@ private static bool IsAnyOS(IEnumerable<string> names)
 
         public static string PathWhich([NotNull] string name)
         {
-            return IsWindows ? PathWhichWindows(name) : PathWhichUnix(name);
+            if (IsWindows)
+            {
+                return PathWhichWindows(name);
+            }
+            return PathWhichUnix(name);
```

Comment:
Why this change? Previous code looked more readable.

Reply:
Do we want to only have Windows vs Unix, or do we want to make this code flexible in case we add more options? The if statement semantically says "check if this is Windows, if so use Windows, and have Unix as a fallback", while the ternary says to me "check if this is Windows, if so use Windows, else this is Unix".

Reply:
I wouldn't really worry about that too much. Supposing this ever needs to change, which is already unlikely, the change is very simple.
That being said, now that the braces were removed this is not that important.